drivers/oled/ssd1306_sh1106.c

```diff
@@ -367,6 +367,8 @@ void oled_advance_page(bool clearPageRemainder) {
     uint16_t index     = oled_cursor - &oled_buffer[0];
     uint8_t  remaining = oled_rotation_width - (index % oled_rotation_width);
 
+    _Static_assert(OLED_FONT_HEIGHT == 8, "OLED buffer implementation assumes a line height of 8 pixels");
```

Comment:
Not really a fan of this, as you *could* use multiples of 8, as well (considering the driver supports 128x64 screens, already)...

Also, this should be separated out into it's own PR, not tossed in with keyboard changes.

Reply:
Multiples of 8 are not currently supported, but I agree that adding support for it would be easier than supporting arbitrary line heights.

I also agree that this change doesn't belong here, but I want to wait a bit with separating it from this PR since this PR could turn into a discussion about the OLED driver. In case the proposed `OLED_IDLE_BLOCKS_MASK` logic is desirable, this entire PR needs to be rewritten and will be more about the OLED driver than about the keyboard changes.

Reply:
I removed the OLED related changes from this PR since I found an approach that does not require them. However, I do think it would be good to document the fact that `OLED_FONT_HEIGHT` is not really supported. Currently, it only says "untested", but really it should say "values other than 8 are not supported". I have no reason to work on the OLED driver anymore, so I'll leave that to those involved (@7-rate?).